boltdesignsystem/bolt
# Bolt Card Component Refactor

## Jira

http://vjira2:8080/browse/BDS-685

## Summary

Enhanced card component to be easily accept free-form content and added whole card clickability feature.

## Details
* Refactored twig templates and CSS to accommodate free-form content
* Web component added
* Whole card clickability is implemented

## How to test

Run the branch locally and navigate to the Card docs page, make sure all demos are doing what they are supposed to do.
